platform/core/src/services/DisplaySetService/DisplaySetService.ts

```diff
@@ -115,16 +115,52 @@ export default class DisplaySetService extends PubSubService {
     return this.activeDisplaySets;
   }
 
+  /**
+   * Gets the set of series with this series instance UID
+   *
+   * <b>WARNING: Do not use this method when you have a referenced series sequence
+   * as this method does NOT check sop instances.  Instead, use getDisplaySetsForReference
+   * to get those with the correct sop instances in them.</b>
+   */
   public getDisplaySetsForSeries = (seriesInstanceUID: string): DisplaySet[] => {
     return [...displaySetCache.values()].filter(
       displaySet => displaySet.SeriesInstanceUID === seriesInstanceUID
     );
   };
 
+  /**
+   * Given a reference to a series/sop, returns the set of display sets
+   * containing an instance from the references.
+   */
+  public getDisplaySetsForReference = (
+    reference: ReferencedSeriesSequence | ReferencedSeriesSequence[]
```

Comment:
strange naming since i see reference can be a list. i recommend references and always make it a list (even with one etnry) much more clear for human and ai